docs/user/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: BTP Operator Module
```

Comment:
```suggestion
title: BTP Operator module
```
We use [sentence case](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#headings) for headings